packages/flutter_tools/lib/src/commands/analyze_base.dart

```diff
@@ -84,7 +85,18 @@ abstract class AnalyzeBase {
     logger.printStatus('Analysis benchmark written to $benchmarkOut ($data).');
   }
 
+  bool get isFlutterRepo => argResults['flutter-repo'] as bool;
+  String get sdkPath => argResults['dart-sdk'] as String ?? artifacts.getArtifactPath(Artifact.engineDartSdkPath);
   bool get isBenchmarking => argResults['benchmark'] as bool;
+  bool get isDartDocs => argResults['dartdocs'] as bool;
+
+  static int countMissingDartDocs(List<AnalysisError> errors) {
+    final int undocumentedMembers = errors.where((AnalysisError error) {
```

Comment:
nit: just return the result of errors.where directly.

```suggestion
    return errors.where((AnalysisError error) {
```